test/listenable_provider_test.dart

```diff
@@ -26,21 +26,6 @@ void main() {
       expect(Provider.of<ChangeNotifier>(key.currentContext, listen: false),
           listenable);
     });
-    testWidgets(
-      'asserts that the created notifier has no listener',
```

Comment:
Is it enough to remove the test for the assertion (which I think is enough), or should I create a dedicated test which tests that a `ListenableProvider` with a listener doesn't fail?

Reply:
Make sure to add a test for this specific use-case. Thanks!

Reply:
Done.